### PR TITLE
OE-5594

### PR DESCRIPTION
**Pull request by vetusko**
Biometry element display in Op. Note

### CODE REVIEW

File: views/default/form_Element_OphTrOperationnote_Biometry.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * @copyright Copyright (c) 2008-2011, Moorfields Eye Hospital NHS Foundation Trust
+ * @copyright Copyright (c) 2011-2013, OpenEyes Foundation
+ * @license http://www.gnu.org/licenses/gpl-3.0.html The GNU General Public License V3.0
+ */
+?>
+
+<?php
+$layoutColumns=$form->layoutColumns;
+$form->layoutColumns=array('label'=>3,'field'=>9);
+// TODO: need to search for the date of the op note to display proper values in the view!!!
```
Comment (petergallagher) [inline views/default/form_Element_OphTrOperationnote_Biometry.php]: Is this now TODONE? Looks like the code exists for this

File: models/Element_OphTrOperationnote_Biometry.php [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+	// because we are working with a view here we do not need to save
+	function __set ($name, $value){
+		return true;
+	}
+
+	function save( $validation = false, $attributes = NULL){
+		return true;
+	}
+}
```
Comment (petergallagher) [inline models/Element_OphTrOperationnote_Biometry.php:133]: These two functions need to have scope declared and save needs an extra param. `Declaration of Element_OphTrOperationnote_Biometry::save() should be compatible with that of BaseActiveRecordVersioned::save()`

File: models/Element_OphTrOperationnote_Biometry.php [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+			'iol_power_left' => 'IOL Power',
+			'predicted_refraction_left' => 'Predicted Refraction',
+			'iol_power_right' => 'IOL Power',
+			'predicted_refraction_right' => 'Predicted Refraction',
+			'target_refraction_left' => 'Target Refraction',
+			'target_refraction_right' => 'Target Refraction',
+		);
+	}
+
+	public function findAll($attributes, $values){
```
Comment (petergallagher) [inline models/Element_OphTrOperationnote_Biometry.php]: This declaration doesn't match the parent.
Declaration of Element_OphTrOperationnote_Biometry::findAll() should be compatible with that of CActiveRecord::findAll()

File: views/default/form_Element_OphTrOperationnote_Biometry.php (hunk 1 above)
Comment (vetusko) [inline views/default/form_Element_OphTrOperationnote_Biometry.php]: yep, just forgot to remove the comment :) :+1:

**Pull request merged**